src/mpcCoreKit.ts

```diff
@@ -387,20 +387,23 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       }
 
       // get postbox key.
-      let loginResponse: TorusKey;
+      let loginPromise: Promise<TorusKey>;
       if (!params.subVerifier) {
         // single verifier login.
-        loginResponse = await this.torusSp.customAuthInstance.getTorusKey(verifier, verifierId, { verifier_id: verifierId }, idToken, {
+        loginPromise = this.torusSp.customAuthInstance.getTorusKey(verifier, verifierId, { verifier_id: verifierId }, idToken, {
           ...params.extraVerifierParams,
           ...params.additionalParams,
         });
       } else {
         // aggregate verifier login
-        loginResponse = await this.torusSp.customAuthInstance.getAggregateTorusKey(verifier, verifierId, [
+        loginPromise = this.torusSp.customAuthInstance.getAggregateTorusKey(verifier, verifierId, [
           { verifier: params.subVerifier, idToken, extraVerifierParams: params.extraVerifierParams },
         ]);
       }
 
+      // wait for prefetch completed before setup tkey
+      const [loginResponse] = await Promise.all([loginPromise, prefetchTssPublicKeys]);
```

Comment:
is there speed improvement ?

Reply:
not really, i think just for code clarify.